UI/OptionsWnd.cpp

```diff
@@ -522,6 +522,7 @@ void OptionsWnd::CompleteConstruction() {
     BoolOption(current_page, 0, "UI.window-quickclose",          UserString("OPTIONS_QUICK_CLOSE_WNDS"));
     BoolOption(current_page, 0, "UI.sidepanel-planet-shown",     UserString("OPTIONS_SHOW_SIDEPANEL_PLANETS"));
     BoolOption(current_page, 0, "UI.auto-reposition-windows",    UserString("OPTIONS_AUTO_REPOSITION_WINDOWS"));
+    BoolOption(current_page, 0, "UI.display-timestamp",          UserString("OPTIONS_DB_DISPLAY_TIMESTAMP"));
```

Comment:
The options DB entry is too verbose for the option label. It works as the tooltip, but the options UI label should be short, like just "Chat Timestamps". Have a separate stringtable entry in the game options section and use that for the extra label...

https://github.com/freeorion/freeorion/blob/master/default/stringtables/en.txt#L2325

Reply:
Fixed.